test/e2e/specs/site-editor/user-global-styles-revisions.spec.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * WordPress dependencies
+ */
+const {
+	test,
+	expect,
+	Editor,
+} = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.use( {
+	editor: async ( { page }, use ) => {
+		await use( new Editor( { page, hasIframe: true } ) );
+	},
+	userGlobalStylesRevisions: async ( { page, requestUtils }, use ) => {
+		await use( new UserGlobalStylesRevisions( { page, requestUtils } ) );
+	},
+} );
+
+test.describe( 'Global styles revisions', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await Promise.all( [
+			requestUtils.activateTheme( 'emptytheme' ),
+			requestUtils.deleteAllTemplates( 'wp_template' ),
+			requestUtils.deleteAllTemplates( 'wp_template_part' ),
+		] );
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
+	} );
+
+	test.afterEach( async ( { requestUtils } ) => {
+		await Promise.all( [
+			requestUtils.deleteAllTemplates( 'wp_template' ),
+			requestUtils.deleteAllTemplates( 'wp_template_part' ),
+		] );
+	} );
+
+	test.beforeEach( async ( { admin } ) => {
+		await admin.visitSiteEditor( {
+			canvas: 'edit',
+		} );
+	} );
+
+	test( 'should display revisions UI when there is more than 1 revision', async ( {
+		page,
+		editor,
+		userGlobalStylesRevisions,
+	} ) => {
+		// Navigates to Styles -> Typography -> Text and click on a size.
+		await page.getByRole( 'button', { name: 'Styles' } ).click();
+
+		const currentRevisions =
+			await userGlobalStylesRevisions.getGlobalStylesRevisions();
+
+		/*
+		 * There are not enough revisions to show the revisions UI yet, so let's create some.
+		 * The condition exists until we have way (and the requirement) to delete global styles revisions.
+		 */
+		if ( currentRevisions.length < 1 ) {
+			// Change a style and save it.
+			await page
+				.getByRole( 'button', { name: 'Typography styles' } )
+				.click();
+			await page
+				.getByRole( 'button', { name: 'Typography Text styles' } )
+				.click();
+			await page.click(
+				'role=radiogroup[name="Font size"i] >> role=radio[name="Large"i]'
```

Comment:
Nit: Why the sudden change of the locator style 😆 ?

Reply:
To make life interesting of course! 😄 
I'll make it consistent according to the chaining tip above. Thank you!